tools/feature_matrix.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+"""Utilities for mapping browser versions to llvm feature flags."""
+
+import logging
+from enum import IntEnum, auto
+
+from .settings import settings
+
+logger = logging.getLogger('feature_matrix')
+
+
+class Feature(IntEnum):
+  NON_TRAPPING_FPTOINT = auto()
+  SIGN_EXT = auto()
+  BULK_MEMORY = auto()
+  MUTABLE_GLOBALS = auto()
+
+
+min_browser_versions = {
+  Feature.NON_TRAPPING_FPTOINT: {
+    'chrome': 75,
+    'firefox': 65,
+    'safari': 150000,
+  },
+  Feature.SIGN_EXT: {
+    'chrome': 74,
+    'firefox': 62,
+    'safari': 141000,
+  },
+  Feature.BULK_MEMORY: {
+    'chrome': 75,
+    'firefox': 79,
+    'safari': 150000,
+  },
+  Feature.MUTABLE_GLOBALS: {
+    'chrome': 74,
+    'firefox': -1, # Firefox has always supported mutable globals apparently
```

Comment:
I found https://bugzilla.mozilla.org/show_bug.cgi?id=1412238, it looks like version 61